nautobot/extras/filters.py

```diff
@@ -734,10 +735,19 @@ class RelationshipAssociationFilterSet(BaseFilterSet):
     destination_type = ContentTypeMultipleChoiceFilter(
         choices=FeatureQuery("relationships").get_choices, conjoined=False
     )
+    peer_id = MultiValueUUIDFilter(method="peer_id_filter")
 
     class Meta:
         model = RelationshipAssociation
-        fields = ["id", "relationship", "source_type", "source_id", "destination_type", "destination_id"]
+        fields = ["id", "relationship", "source_type", "source_id", "destination_type", "destination_id", "peer_id"]
+
+    def peer_id_filter(self, queryset, name, value):
+        filtered_value = [entry.id for entry in queryset if entry.relationship.symmetric]
+        # Filter down to symmetric relationships only.
+        queryset = queryset.filter(id__in=filtered_value)
```

Comment:
Could we do something like `queryset = queryset.filter(relationship__type__in=[RelationshipTypeChoices.TYPE_ONE_TO_ONE_SYMMETRIC, RelationshipTypeChoices.TYPE_MANY_TO_MANY_SYMMETRIC])` rather than the separate list calculation?